File: Aplikacija/FE/Diplomski/src/app/models/producerDTO.ts
export interface Producer
{
    id : string,
    name : string,
    description? : string
}

export interface ProducerInfo
{
    name : string,
    description? : string
}

export interface CreateProducerDto
{
    name : string,
    description? : string
}


export interface UpdateProducerDto
{
    name : string,
    description? : string
}

File: Aplikacija/BE/DiplomskiRAD/DTOs/ProducerDTO.cs
namespace DiplomskiRAD.DTOs
{
    public class ProducerDTO
    {
        public class ProducerInfo
        {

            public string Name { get; set; }
            public string? Description { get; set; }

            public ProducerInfo(string name, string? description)
            {
                Name = name;
                Description = description;
            }

            public ProducerInfo() { }

            public ProducerInfo(string name)
            {
                Name = name;
            }
        }

        public class CreateProducerDto
        {
            public string Name { get; set; }
            public string? Description { get; set; }
        }

        public class UpdateProducerDto
        {
            public string Name { get; set; }
            public string? Description { get; set; }
        }

    }
}
